tests/network/vmi_passt.go

```diff
@@ -67,7 +67,7 @@ var _ = SIGDescribe("[Serial] Passt", decorators.PasstGate, Serial, func() {
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
 			)
 
-			vmi, err = virtClient.VirtualMachineInstance(testsuite.NamespacePrivileged).Create(context.Background(), vmi)
+			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
```

Comment:
It looks odd to me that to get the default namespace for tests we need to call a function and pass nil to it.
Cant we just use the `NamespaceTestDefault` instead?

Reply:
This conventions is used in other tests, for example in:
https://github.com/kubevirt/kubevirt/blob/3bee1de7b4f8bab059817cf44d636d2521dec8a9/tests/network/vmi_multus.go#L167

I don't mind changing it to `NamespaceTestDefault`, but I think there should be a policy regarding which namespace to use.

Reply:
@AlonaKaplan WDYT?

Reply:
Please follow the convention